### PR TITLE
Add optional command line option for config file

### PR DESCRIPTION
**Pull request by BafDyce**
Implements #28

### CODE REVIEW

**Comment by BafDyce** [comment]
@fsteffek @DaChrissi 
Please do a quick source code check and some testing of the new functionaility (whatever you deem necessary).

File: mumbleBot.py [hunk 1]
```diff
@@ -332,7 +329,11 @@ def start_web_interface(addr, port):
 
 
 if __name__ == '__main__':
-    parser = argparse.ArgumentParser(description='Bot for playing radio stream on Mumble')
+    global __CONFIG
+    parser = argparse.ArgumentParser(description='Bot for playing music on Mumble')
```
Comment (fsteffek) [inline mumbleBot.py]: Changing `radio stream` to `music` has nothing to do with the submitted feature. Please commit in another PR. Thanks 😃

**Comment by BafDyce** [comment]
I've found a severe bug with this PR: `__CONFIG` is not available in `utils.py` due to variable scope issues. I'm working on a fix for that.

File: mumbleBot.py (hunk 1 above)
Comment (BafDyce) [reply mumbleBot.py]: Fixed

**Comment by BafDyce** [comment]
Yeah, I would love to get rid off them too but I don't have a better solution for it neither.

**Pull request merged**